environments/prod/prod.tfvars

```diff
@@ -2673,4 +2673,18 @@ frontends = [
       }
     ]
   },
+  {
+    product        = "et"
+    name           = "et-sya"
+    mode           = "Detection"
+    custom_domain  = "claim-employment-tribunals.service.gov.uk"
```

Comment:
You need to add `certificate_name` attribute here like https://github.com/hmcts/azure-platform-terraform/blob/526a2399ea19cdad28f0e12893adf395e3e40eb7/environments/prod/prod.tfvars#L1502

As you need a custom certificate you can find out how to do that here :
https://hmcts.github.io/information-security/certificate-automation.html#manual-certificate-requests

Reply:
Certificate added